docs/javascripts/site.js

```diff
@@ -1,92 +1,101 @@
-// Function to initialize DataTables
-function initializeDataTables() {
-    if (!$.fn.DataTable.isDataTable('#test_table')) {
-        $('#test_table').DataTable({
-            pageLength: -1,
-            scrollX: true,
-            autoWidth: false
-        });
-    }
-}
+// Config
+const FILTER_INPUT_SELECTOR = ".custom_dt_filter";
+let table;
 
-// Function to attach event listeners to the selectors
-function attachFilterListeners() {
+// Subscribe to when a page is loaded in mkdocs.
+// see: https://github.com/squidfunk/mkdocs-material/issues/5816#issuecomment-1667654560
+document$.subscribe(() => {
```

Comment:
This breaks the data table initialization in the standalone pages as `site.js` also gets included there:
https://github.com/raxhvl/execution-spec-tests/blob/0c2549d6fc42a59b6a338e3f8821da722ce95827/docs/templates/function.html.j2#L24